### PR TITLE
httpstream cleanups

### PR DESCRIPTION
**Pull request by mbouron**
Second wave of cleanups, this time regarding the vlc backend.

### CODE REVIEW

**Comment by CDrummond** [comment]
Not sure I agree with the removal of the vlc timer - this was to work-around https://github.com/CDrummond/cantata/issues/987 The status is only updated every 5 seconds.

**Comment by mbouron** [comment]
OK. I removed this commit from the PR for now and see if we can avoid the use of a timer later on.

**Comment by CDrummond** [comment]
Thanks. The timer /might/ not be required - I just know it was put in there to solve 1 specific issue. However, I'm starting to think I should just remove the libVLC support - its not the default anymore.

File: mpd-interface/httpstream.cpp [hunk 1]
```diff
@@ -131,7 +131,7 @@ void HttpStream::toggleMute()
 {
     DBUG << isMuted();
     if (player) {
-        muted=!muted;
+        muted =! muted;
```
Comment (cpitclaudel) [inline mpd-interface/httpstream.cpp]: This is really weird spacing. Shouldn't it be ` = !muted`?

**Pull request merged**